### DISCUSSION TITLE
Kafka incompatible with Chip M1 Go

### DISCUSSION THREAD

**Discussion by eestradag** [confluentinc/confluent-kafka-go#825] [Q&A]
Dear, team

I have a problem, I need build my proyect but show me an error.  but other persons of my teams have PC linux and the build es success, but I have a MacBook M1
and when i try to build the proyect, this show me this message. I search for internet and any forums say that it's
a problem with the chip. Can you help me please.
github.com/confluentinc/confluent-kafka-go@v1.9.1/kafka/librdkafka_vendor/librdkafka_musl_linux.a(rdkafka_error.o)

Thank!!

**Comment by holgerson97** [comment]
I am using github.com/confluentinc/confluent-kafka-go v1.9.3, also on MacBook M1. For me it works perfectly. What is your error message?
